src/main/java/duke/command/AddCommand.java

```diff
@@ -1,3 +1,11 @@
+package duke.command;
+
+import duke.constant.TaskType;
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.*;
+import duke.ui.Ui;
+
 public class AddCommand extends Command {
```

Comment:
Maybe addCommand should be a method within Command rather than a class on it's own?